_templates/hello.html

```diff
@@ -8,11 +8,11 @@
       {{ name }}
   </div>
 
-  <div class="major">
+  <div class="Mathemathics">
       {{ major }}
   </div>
 
-  <div class="year">
+  <div class="2027">
```

Comment:
these are the class names, not content. 

the `{{ major }}` syntax pulls content from your `info.yml` file.